app/containers/message/Video.js

```diff
@@ -41,18 +55,44 @@ const Video = React.memo(({
 		openLink(uri, theme);
 	};
 
+	const fetchThumbnail = async() => {
+		if (isTypeSupported(file.video_type)) {
+			const uri_ = formatAttachmentUrl(file.video_url, user.id, user.token, baseUrl);
+
+			const { uri } = await VideoThumbnails.getThumbnailAsync(
+				uri_
+			);
+			setImage(uri);
+		}
+	};
+	useEffect(() => {
+		fetchThumbnail();
+	}, []);
 	return (
 		<>
 			<Touchable
 				onPress={onPress}
-				style={[styles.button, { backgroundColor: themes[theme].videoBackground }]}
+				style={[styles.button]}
 				background={Touchable.Ripple(themes[theme].bannerBackground)}
 			>
-				<CustomIcon
-					name='play-filled'
-					size={54}
-					color={themes[theme].buttonText}
-				/>
+				<View>
+					<FastImage
+						source={{ uri: image }}
+						style={styles.thumbnailStyles}
+						resizeMode='contain'
```

Comment:
As mentioned earlier, this might look better on the chat if we set `resizeMode` to either `cover` or `center`.